src/utils/data.js

```diff
@@ -1,4 +0,0 @@
-export const flatten = list =>
-  list.reduce((a, b) => a.concat(Array.isArray(b) ? flatten(b) : b), []);
-
-export const echo = list => list;
```

Comment:
There is lodash (https://lodash.com/docs/) identity function that can be used to achieve this too.
Lodash is a nice library - it pretty much has all the little helper functions that javascript natively did not have or still does not have. I was a late convert but now use it often.
The only thing to keep in mind is that number of its helpers are also now native with es6.